_docs/static-setup.md

```diff
@@ -1,26 +1,119 @@
 ---
-title: "Adding yourself to the team page"
-sidebar-title: "Add to Team Page"
+title: "Website Setup"
+sidebar-title: "Website Setup"
 ---
 
+# Website Setup: In-Depth Walkthrough
+{:.no_toc}
+
 ## Table of Contents
 {:.no_toc}
 * TOC
 {:toc}
 
-## Get ready
+## One-Time Setup
+
+These next steps take a few minutes of time, but you *only need to do them once*.
+
+### Prerequisites
+
+Before we get started, make sure that you've already followed along with our [dev setup guide]({{ site.baseurl }}/docs/dev-setup) to get your terminal, git, and GitHub set-up and to understand what all the `git` commands we'll run do! From here, we'll walk you through step-by-step on the components necessary to work on our website and how to set up each component. We'll tell you how to check if you already have each component, how to briefly install it if you don't, and then show you some extra resources where you can learn more (but only if you want)!
 
-Before we get started, we highly encourage that you follow along with our [dev setup guide]({{ site.baseurl }}/docs/dev-setup) to get a working development environment set up. From here, there's just a few commands that we'll need to execute in sequence to get things rolling. This should be review:
+You may ask: why is the website so complicated to set up! You may have a point, but it allows us to write features that let you update the website **without writing code**: we've written templates that generate pages, content, and complicated data relationships, so you don't have to! Hopefully, with practice, you'll find this to be not too tricky.
+
+### Cloning Our Repository
+
+You can review what `git clone` does in our [Dev Setup Guide]("{{site.baseurl}}/docs/dev-setup") but if you want to get our website's repository set-up on our machine you can just run the following `git` command:
 
 ```sh
-$ git checkout -b my-branch-name    # create a new branch and check it out
-$ jekyll serve                      # set up our dev server
+$ git clone https://github.com/uclaacm/teach-la-website.git
```

Comment:
Ooh, I'd use the ssh clone by default if possible. Might be good to recommend the ssh keygen stuff as well in the dev setup.